examples/integration/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/CoreUtilsObjC.xcscheme

```diff
@@ -15,7 +15,7 @@
                   <BuildableReference
                      BuildableIdentifier = "primary"
                      BlueprintIdentifier = "64AE6000A6317B9C077F3B1E"
-                     BuildableName = "CoreUtilsObjC"
+                     BuildableName = "CoreUtilsObjC.framework"
```

Comment:
Not sure on why this was the other `BuildableName`, this is correct now at least.